scrapers/LittleCapriceDreams.yml

```diff
@@ -0,0 +1,24 @@
+name: LittleCaprice
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.littlecaprice-dreams.com/project
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@property="og:title"]/@content
+      Details: //meta[@property="og:description"]/@content
+      Image: //meta[@property="og:image"]/@content
+      Studio:
+        Name: //meta[@property="og:site_name"]/@content
+      Date:
+        selector: //div[@class="vid_date"]
+        parseDate: January 2, 2006
+      Performers:
+        Name: //div[@class="vid_info_content"][1]//a
+driver:
```

Comment:
why is useCDP needed ?
it works ok without it.

Reply:
You are correct, it works without it. In my initial testing when I was attempting to scrap several fields and it seemed to not work without the useCDP. I should have tested the final without the useCDP. That was an oversight on my part. Let's pull that out.